docs/source/json-api/index.rst

```diff
@@ -93,6 +93,21 @@ Start HTTP service from a DAML project directory
             prefix -- URL prefix,
             directory -- local directory that will be mapped to the URL prefix.
             Example: "prefix=static,directory=./static-content"
+      --access-token-file <value>
+        provide the path from which the access token will be read, required to interact with an authenticated ledger, no default
+
+With Authentication
+===================
+
+If you are running the HTTP JSON API against a Ledger API server that requires authentication, you must provide the access token when you start it.
```

Comment:
The last `it` may refer to either the _HTTP JSON API_ or the _Ledger API Server_

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3657/commits/1bfdf8281ba0f599cc4a87d595f2afd120f8c413